addons/pos_adyen/models/pos_payment_method.py

```diff
@@ -26,6 +26,13 @@ def _get_payment_terminal_selection(self):
     adyen_test_mode = fields.Boolean(help='Run transactions in the test environment.', groups='base.group_erp_manager')
 
     adyen_latest_response = fields.Char(copy=False, groups='base.group_erp_manager') # used to buffer the latest asynchronous notification from Adyen.
+    adyen_event_url = fields.Char(
+        string="Event URL",
+        help="This URL needs to be pasted on Adyen's website, in Event URLs configuration",
```

Comment:
```suggestion
        help="This URL needs to be pasted on Adyen's portal terminal setting, see: https://www.odoo.com/documentation/master/applications/sales/point_of_sale/payment_methods/terminals/adyen.html#set-the-event-urls",
```

Reply:
We can't copy a link from here, this is an hover effect which disappears when you try to hover the text.